### ISSUE TITLE
pv size automated expansion supported by this controller?

### ISSUE THREAD

**Issue by aakkiiff** [kubernetes-sigs/nfs-subdir-external-provisioner#356]
i wanted to expand the pv size ,but the controller can not expand it.
is there any way to automate this expansion .
or any manual way?

**Comment by niranjandarshann** [comment]
@aakkiiff  yes you can automate PV expansion using cronjob. If your CSI driver does not support expansion.

**Comment by niranjandarshann** [comment]
Some storage backend like nfs hostPath do not support volume expansion. you can manually resize them manually resize them at the storage level by increasing EBS volume size on AWS, resizing a disk on GCP/Azure, or modifying the NFS export.
